README.md

```diff
@@ -6,24 +6,16 @@
 
 ## Set up with Docker
 
-1. Create a `local_settings.py` file in the project folder:
-   * use the `local_settings.py.docker.template`
-   * set `DEBUG` the way you need it
+1. Create a `docker-compose.env.yaml` file in the project folder:
+   * you can use `local_settings.py.docker.template` as a base, it does not need any changes if you want all features enables
```

Comment:
do you mean `docker-compose.env.yaml.template`?

Reply:
That I do indeed :)